### PR TITLE
Add compat data for FetchEvent (Fetch API)

### PR DESCRIPTION
**Pull request by maboa**
I'm not sure I represented [`respondWith`](https://developer.mozilla.org/en-US/docs/Web/API/FetchEvent/respondWith) correctly. Please feel free to suggest a better way - thanks! :)

### CODE REVIEW

File: api/FetchEvent.json [hunk 1]
```diff
@@ -0,0 +1,439 @@
...
+            }
+          }
+        }
+      },
+      "client": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/FetchEvent/client",
+          "support": {
+            "webview_android": {
+              "version_added": null
```
Comment (jpmedley) [inline api/FetchEvent.json]: Hold off on merging for now. I don't think this is null. Let me see if I can get an answer.

Comment (jpmedley) [reply api/FetchEvent.json]: I forgot this was a deprecated feature. So, never mind.

File: api/FetchEvent.json [hunk 2]
```diff
@@ -0,0 +1,439 @@
...
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/FetchEvent/isReload",
+          "support": {
+            "webview_android": {
+              "version_added": null
+            },
+            "chrome": {
+              "version_added": "40"
+            },
+            "chrome_android": {
+              "version_added": "44"
```
Comment (jpmedley) [inline api/FetchEvent.json]: All 3 Chrome versions should be 45.

File: api/FetchEvent.json [hunk 3]
```diff
@@ -0,0 +1,439 @@
...
+            },
+            "ie": {
+              "version_added": false
+            },
+            "opera": {
+              "version_added": "24"
+            },
+            "opera_android": {
+              "version_added": null
```
Comment (jpmedley) [inline api/FetchEvent.json]: Both Opera versions should be 32.

File: api/FetchEvent.json [hunk 4]
```diff
@@ -0,0 +1,439 @@
...
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/FetchEvent/request",
+          "support": {
+            "webview_android": {
+              "version_added": null
+            },
+            "chrome": {
+              "version_added": "40"
+            },
+            "chrome_android": {
+              "version_added": "44"
```
Comment (jpmedley) [inline api/FetchEvent.json]: `FetchEvent/request` should just be `true` for Chrome and Opera. I can't pin down a version number.

File: api/FetchEvent.json [hunk 5]
```diff
@@ -0,0 +1,439 @@
...
+              "version_added": "40",
+              "notes": "NetworkError thrown if request mode is same-origin and response type is cors (see <a href='https://bugzil.la/1222008'>bug 1222008</a>). This is being worked on — see <a href='https://www.chromestatus.com/feature/5694278818856960'>https://www.chromestatus.com/feature/5694278818856960</a>."
+            },
+            "chrome": {
+              "version_added": "40",
+              "notes": "NetworkError thrown if request mode is same-origin and response type is cors (see <a href='https://bugzil.la/1222008'>bug 1222008</a>). This is being worked on — see <a href='https://www.chromestatus.com/feature/5694278818856960'>https://www.chromestatus.com/feature/5694278818856960</a>."
+            },
+            "chrome_android": {
+              "version_added": "40",
+              "notes": "NetworkError thrown if request mode is same-origin and response type is cors (see <a href='https://bugzil.la/1222008'>bug 1222008</a>). This is being worked on — see <a href='https://www.chromestatus.com/feature/5694278818856960'>https://www.chromestatus.com/feature/5694278818856960</a>."
```
Comment (jpmedley) [inline api/FetchEvent.json]: These should be 42 for all Chrome and 29 for all Opera. https://www.chromestatus.com/features/6730533392351232

File: api/FetchEvent.json (hunk 1 above)
Comment (maboa) [reply api/FetchEvent.json]: 👍

File: api/FetchEvent.json [hunk 6]
```diff
@@ -0,0 +1,439 @@
...
+            "deprecated": true
+          }
+        }
+      },
+      "clientId": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/FetchEvent/clientId",
+          "support": {
+            "webview_android": {
+              "version_added": null
```
[the remark this answers is not in the document]
Comment (maboa) [reply api/FetchEvent.json]: 👍

File: api/FetchEvent.json [hunk 7]
```diff
@@ -0,0 +1,439 @@
...
+              "notes": "Service workers (and <a href='https://developer.mozilla.org/docs/Web/API/Push_API'>Push</a>) have been disabled in the <a href='https://www.mozilla.org/en-US/firefox/organizations/'>Firefox 45 and 52 Extended Support Releases</a> (ESR.)"
+            },
+            "firefox_android": {
+              "version_added": "45"
+            },
+            "ie": {
+              "version_added": false
+            },
+            "opera": {
+              "version_added": null
```
[the remark this answers is not in the document]
Comment (maboa) [reply api/FetchEvent.json]: 👍

File: api/FetchEvent.json [hunk 8]
```diff
@@ -0,0 +1,439 @@
...
+            },
+            "opera_android": {
+              "version_added": null
```
[the remark this answers is not in the document]
Comment (maboa) [reply api/FetchEvent.json]: :+1:

File: api/FetchEvent.json (hunk 2 above)
Comment (maboa) [reply api/FetchEvent.json]: 👍

File: api/FetchEvent.json (hunk 3 above)
Comment (maboa) [reply api/FetchEvent.json]: 👍

File: api/FetchEvent.json (hunk 4 above)
Comment (maboa) [reply api/FetchEvent.json]: 👍

File: api/FetchEvent.json (hunk 5 above)
Comment (maboa) [reply api/FetchEvent.json]: 👍

**Comment by maboa** [comment]
Pushed changes in line with review.

File: api/FetchEvent.json [hunk 9]
```diff
@@ -0,0 +1,439 @@
...
+        },
+        "status": {
+          "experimental": true,
+          "standard_track": true,
+          "deprecated": false
+        }
+      },
+      "preloadResponse": {
+        "__compat": {
+          "description": "<code>preloadResponse</code> property",
```
[the remark this answers is not in the document]
Comment (maboa) [reply api/FetchEvent.json]: OK. Have assumed the same status as the other properties.

File: api/FetchEvent.json [hunk 10]
```diff
@@ -0,0 +1,439 @@
...
+            }
+          },
+          "status": {
+            "experimental": true,
+            "standard_track": true,
+            "deprecated": false
+          }
+        }
+      },
+      "resource_url": {
```
[the remark this answers is not in the document]
Comment (maboa) [reply api/FetchEvent.json]: Done. Thanks - looks like a useful resource.

**Pull request merged**